src/v2/Components/BankDebitForm/BankDebitProvider.tsx

```diff
@@ -0,0 +1,71 @@
+import { FC, useEffect, useState } from "react"
+import { Appearance, loadStripe } from "@stripe/stripe-js"
+import { Elements } from "@stripe/react-stripe-js"
+import { getENV } from "v2/Utils/getENV"
+import { commitMutation, graphql } from "relay-runtime"
+import { createBankDebitSetupMutation } from "v2/__generated__/createBankDebitSetupMutation.graphql"
+import { useSystemContext } from "v2/System"
+import { BankDebitForm } from "./BankDebitForm"
+
+export const BankDebitProvider: FC = () => {
+  const { relayEnvironment } = useSystemContext()
+  const [clientSecret, setClientSecret] = useState("")
+
+  const stripePromise = loadStripe(getENV("STRIPE_PUBLISHABLE_KEY"), {
```

Comment:
[Here](https://stripe.com/docs/stripe-js/react#elements-provider) mentioned that we might want to move `loadStripe` outside to avoid recreating the `Stripe` object on every render.

Reply:
I tried this and it broke the `mockStripe` tests for some reason. I will look into it more.

Reply:
Ah, I forgot to look into this, mind if I make this a task for the next PR?